lib/Bio/EnsEMBL/DataCheck/Test/DataCheck.pm

```diff
@@ -421,4 +422,36 @@ sub denormalised {
   return denormalized(@_);
 }
 
+=head2 Testing empty values  
+
+=over 4
+
+=item B<no_missing_value>
+
+no_missing_value($dbc, $table, $column, $id, $test_name, $diag_msg);
+
+Tests if a C<$table> contains C<$column> with null or empty values 
+by calling B<is_rows_zero>. Which means, if the number of 
+rows is zero the test will pass. The C<$id> will be used for 
+a C<SELECT> statement to count the rows that are returned and print 
+them as a diagnostic message. 
+
+=back
+
+=cut
+
+sub no_missing_value {
```

Comment:
The function looks good, but I'm not a big fan of the name; it's sort of double-negative that takes a few beats to work out what it is testing. Perhaps `has_values` instead? I don't love that either, so feel free to argue/suggest something else.

Reply:
I tried to follow the same logic as 'is_rows_zero' - if number of rows zero the test pass. So I thought the name should say 'no_missing_value' - if there are no missing values the test pass. If 'no_missing_value' is too confusing, could it be 'missing_value'? The name 'has_values' is also another possibility as you suggested.

Reply:
Thanks, I appreciate your arguments, in the end I decided to go with 'has_data'.